app/sidekiq/lighthouse/submit_career_counseling_job.rb

```diff
@@ -10,10 +12,29 @@ class SubmitCareerCounselingJob
     sidekiq_options retry: RETRY
 
     sidekiq_retries_exhausted do |msg, _ex|
-      Rails.logger.error(
-        "Failed all retries on SubmitCareerCounselingJob, last error: #{msg['error_message']}"
-      )
-      StatsD.increment("#{STATSD_KEY_PREFIX}.exhausted")
+      begin
+        claim = SavedClaim.find(msg['args'].first)
+      rescue
+        claim = nil
+      end
+
+      pcpg_monitor = PCPG::Monitor.new
+      pcpg_monitor.track_submission_exhaustion(msg, claim)
+
+      if Flipper.enabled?(:pcpg_trigger_action_needed_email)
+        email = claim.parsed_form.dig('claimantInformation', 'emailAddress')
```

Comment:
Put this block of code into it's own method for clarity

Reply:
changes made!